package.json

```diff
@@ -28,8 +28,8 @@
   "license": "Apache-2.0",
   "private": false,
   "peerDependencies": {
-    "ember-template-lint": "^2.10.0",
-    "prettier": "^1.18.2"
+    "ember-template-lint": ">=2.10.0",
```

Comment:
was it intentional that this will also target e.g. ember-template-lint v42.3.1?

Reply:
yeah, sounds reasonable. just wasn't sure if it was intentional since the PR description was empty 😅